docs/docs/api/Tasks/createTasks.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Create Task
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>POST tasks/</Endpoint>: Create a task with provided task details <br></br>
+
+```json
+{
+    "title": string,
+    "user_id": string,
+    "description": string,
+    "startDate": string,
+    "endDate": string,
+    "isStarted": boolean
+}
+```
+
+### Example Request
+
+This is a **protected route**, a **valid JWT is required** in the header field
+
+#### Header
+
+```
+Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
+```
+
+#### Body
+
+```json
+{
+  "title": "Task 2 - evvvve",
+  "user_id": "62227aab7a9965d0be73b052",
+  "description": "john",
+  "startDate": "2022-02-25T15:02:08",
+  "endDate": "2022-02-27T15:02:08",
+  "isStarted": false
+}
+```
+
+**Note**: Every attribute is required <br></br>
+
+### Possible Responses
+
+#### Immediate Success
+
+```json
+{
+  "title": "Task 2 - evvvve",
+  "user_id": "62227aab7a9965d0be73b052",
+  "description": "john",
+  "startDate": "2022-02-25T20:02:08.000Z",
+  "endDate": "2022-02-27T20:02:08.000Z",
+  "isStarted": false,
+  "_id": "623688c4f5735bc51286682f",
+  "createdAt": "2022-03-20T01:52:04.564Z",
+  "updatedAt": "2022-03-20T01:52:04.564Z",
+  "__v": 0
+}
+```
+
+#### Failure
+
+```json
+{
+    "message": "Task validation failed: user_id: Cast to ObjectId failed for value \"62227aab7a99673b052\" (type string) at path \"user_id\"",
+    "stack": "ValidationError: Task validation failed: user_id: Cast to ObjectId failed for value \"62227aab7a99673b052\" (type string) at path \"user_id\"
```

Comment:
Just a side note: Although this is fine for development, as we start to clean things up, we'll need to remove this as having an ObjectId being revealed can pose some security risks